e2e/testcafe-devextreme/helpers/restoreBrowserSize.ts

```diff
@@ -2,9 +2,9 @@
 type BrowserSizeType = [width: number, height: number];
 const DEFAULT_BROWSER_SIZE: BrowserSizeType = [1200, 800];
 
-const restoreBrowserSize = async (t: TestController):
+const restoreBrowserSize = async (t: TestController, sizeRestoreTo?: BrowserSizeType):
```

Comment:
I don't think it's ok you're specifying non-default size for form test on runner level and then extend `restoreBrowserSize` to restore it to a non-default size manually